core/controllers/collection_editor.py

```diff
@@ -102,20 +104,40 @@ def get(self, collection_id):
         collection = collection_services.get_collection_by_id(
             collection_id, strict=False)
 
+        can_edit = (
+            bool(self.user_id) and
+            self.username not in config_domain.BANNED_USERNAMES.value and
```

Comment:
That's a good question. I think keeping the existing checks is fine, except that you can replace the 'banned' check with the 'whitelist' check.